scalding-core/src/main/scala/com/twitter/scalding/Config.scala

```diff
@@ -267,8 +267,8 @@ trait Config {
    */
   def addReducerEstimator(clsName: String): Config =
     update(Config.ReducerEstimators) {
-      case None => Some(clsName) -> ()
-      case Some(lst) => Some(clsName + "," + lst) -> ()
+      case None => Some(clsName) -> Unit
```

Comment:
This does not seem like an error to me. `Unit` is the type or the companion object where as `()` is the value of type `Unit`. This code is different, although it does not matter because we discard the side value at the end. 

Why the change?